server/src/services/cfadock.service.ts

```diff
@@ -7,8 +7,8 @@ export const CFADOCK_FILTER_LIMIT = 100
 const getSirenParams = (sirenSet: Set<string>): any[] => {
   const sirenParams = []
 
-  sirenSet.forEach((v) => {
-    sirenParams.push({ siret: v })
+  sirenSet.forEach((siret) => {
+    sirenParams.push({ siret })
   })
```

Comment:
La version "one-liner" :)
```suggestion
  sirenSet.forEach((siret) => sirenParams.push({ siret }))
```